src/book/02-system/04-other/01-nubook.mdx

```diff
@@ -5,3 +5,275 @@ title: NUbook
 description: The NUbots team handbook and documentation.
 slug: /system/nubook
 ---
+
+NUbook is the handbook and high-level documentation for the NUbots team. What you are looking at right now is NUbook! This page has information on how NUbook works.
+
+## Getting Started
+
+### Install Prerequisites
+
+You'll need [Git](https://git-scm.com/), [Node.js](https://nodejs.org/en/) and [Yarn](https://classic.yarnpkg.com/en/) installed.
+
+<details>
+<summary>Ubuntu installation instructions</summary>
+
+1. Install Git by running the following commands:
+
+   ```
+   sudo apt update
+   sudo apt install git
+   ```
+
+2. Install the latest version of Node.js by following the instructions [here](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions) to (pick the latest version).
+3. Install the latest version of Yarn by following the instructions [here](https://classic.yarnpkg.com/en/docs/install#debian-stable).
+
+</details>
+
+<details>
+<summary>Windows installation instructions</summary>
+
+1. Download and install Git from <https://git-scm.com/download/win>
+2. Download and install Node.js from <https://nodejs.org/en/> (pick the LTS version)
+3. Download and install Yarn from <https://classic.yarnpkg.com/latest.msi>
+
+</details>
+
+<details>
+<summary>macOS installation installation</summary>
+
+1. Install Homebrew by following the instructions at <https://brew.sh/>
+2. Install Git, Node.js and Yarn
+
+   ```sh
+   brew upgrade
+   brew install git node && brew install yarn
+   ```
+
+</details>
+
+### Install NUbook
+
+1. Navigate to the directory where you want to install NUbook
+
+   ```sh
+   cd <path>
+   ```
+
+2. Clone this repo and `cd` into the cloned directory
+
+   ```sh
+   git clone https://github.com/NUbots/NUbook.git
+   cd NUbook
+   ```
+
+3. Install dependencies
+
+   ```sh
+   yarn
+   ```
+
+4. Run the [Gatsby](https://www.gatsbyjs.org/) development server
+
+   ```sh
+   yarn dev
+   ```
+
+5. Preview the site by visiting [localhost:8000](http://localhost:8000) in a browser
+
+## Contributing Content
+
+All additions and edits are done through GitHub pull requests. To add or edit content:
+
+1. Clone the repo, install dependencies, and run the development server as shown in [Getting started](#getting-started) above.
+2. Create a new branch for your changes, in the format `your_surname/short_description_of_change`:
+
+   ```sh
+   git checkout -b paye/add_2019_debrief
+   ```
+
+3. Make your changes by adding or editing MDX files in [`src/book/`](src/book/). See below for how to write and organise pages.
```

Comment:
Should we specify here that you can add or edit other files, rather than just MDX?

Reply:
For contributing content to NUbook, it's pretty much MDX and image files, which are described further down the page.

Reply:
true. I was thinking more about the `.js` files in the root or something, but that's not "content" now I think about it